docker/deployed/mainnet/healthbot/config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Healthbot won't be running in `mainnet`, so this file is just being coherent with the overall configuration setup.

In any case, we'll soon deprecate relaying in all envs, making this less relevant. 
I think healthbot will still exist, but it won't need the `Chains` section anymore and might be only doing read-queries as probes. (To be discussed at some point in the future).